CHANGELOG.md

```diff
@@ -9,20 +9,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 #### Features
 
-- Add optional PodDisruptionBudget resource for Vector (#57)
+- Add optional PodDisruptionBudget resource for Vector (#57) ([ea65bd0](ea65bd095cb876b29a747954fdaef28370d87a22))
+- Support injecting multiple ConfigMaps (#58) ([337d7d2](337d7d231ee1d307987fe4945a37bfa6f44c5342))
+  - **BREAKING**: Renamed `existingConfigMap` to `existingConfig` and `haproxy.existingConfigMap` to `haproxy.existingConfig`
```

Comment:
Alternatively just put the **BREAKING** in line with the commit message which may not have the exact details 🤔

Reply:
I think this is good as a description.

Reply:
👍 just need to make sure going forward `BREAKING CHANGE: this is a breaking change` is added to the squash/merge